### PR TITLE
Range slider

### PR DESCRIPTION
**Pull request by ivanmalagon**
#94 
---
6-Aug-2018

Copypasting from @jesusbotella comment

---

I have just pushed the latest changes of RangeSlider. There are 2 things missing, setting aria attributes (aria-valuemax, ariavaluemin…) to <as-range-slider> tag and apply step transforming function to slider values (you can find an example function in React’s slider transformer.js file). Everything apart from that should be working OK.

There’s some logic which can be extracted to auxiliar methods, and SCSS might need some variables over there, but it should be OK :slightly_smiling_face:

---

https://deploy-preview-141--carto-airship.netlify.com/catalog/#/components/as-range-slider

### CODE REVIEW

**Comment by VictorVelarde** [comment]
✨ Everything LGTM in your latest changes @jesusbotella 
Great improvement in bar movement and specs!

File: examples/layouts/basic-layout.html [hunk 1]
```diff
@@ -44,4 +44,4 @@
   <script defer src="./common/map.js"></script>
 </body>
 
-</html>
+</html>
```
Comment (IagoLast) [inline examples/layouts/basic-layout.html]: 👀

Comment (IagoLast) [reply examples/layouts/basic-layout.html]: same issue multiple files

Comment (jesusbotella) [reply examples/layouts/basic-layout.html]: I think @VictorVelarde has a preset to remove trailing blank lines from files :(

I am gonna revert those changes.

Comment (VictorVelarde) [reply examples/layouts/basic-layout.html]: Sorry, I'll check it out. My user settings includes `"files.trimTrailingWhitespace": false` but maybe it is some other extension

Comment (VictorVelarde) [reply examples/layouts/basic-layout.html]: I've changed my VSCode user settings to this:
```
 "files.trimTrailingWhitespace": true,
 "files.insertFinalNewline": true,
 "files.trimFinalNewlines": true,
```
This guarantees one, and only one, new line at EOF

**Pull request merged**